docs/argocd-gitops/onboarding_to_argocd.md

```diff
@@ -42,6 +61,8 @@ spec:
   destinations:
     - namespace: '<team-namespace-prefix>-*'
       server: '<cluster-name>'
+    - namespace: '<namespace-name>' # if there is no generic team name prefix, then specify namespaces individually
+      server: '<cluster-name>'
```

Comment:
should be `name` , or the description for `<cluster-name>` should change

Reply:
good catch, updated!